database/test_data.py

```diff
@@ -570,9 +599,53 @@
         "user": "khatts",
         "hoursEarned": 1.0
     },
+
 ]
 CourseParticipant.insert_many(courseHoursEarned).on_conflict_replace().execute()
 
+
+# course = ForeignKeyField(Course)
+# questionContents = CharField() # Check this one
+# questionNumber = IntegerField()
+
+courseQuestions = [
+    {
+    "course":1,
+    "questionContent":" Why are you interested in teaching this course?",
+    "questionNumber":1,
+    },
+    {
+    "course":1,
+    "questionContent":"Is there anything confusing?",
+    "questionNumber":2,
+    },
+    {
+    "course":1,
+    "questionContent":"How many students willl betaking this course?",
+    "questionNumber":3,
+    },
+    {
+    "course":1,
+    "questionContent":" This is another random question",
+    "questionNumber":4,
+    },
+    {
+    "course":1,
+    "questionContent":" Why are you interested in teaching this course?",
+    "questionNumber":5,
+    }
+]
```

Comment:
Make sure you have more than one course in there, or else you can't properly test that your code handles all cases.

Reply:
Though I just noticed that there's not functionality with this change, so I'm ok with leaving it for now, and adding more data when there's something to test.